katcp/server.py

```diff
@@ -2318,6 +2351,79 @@ def inform_callback(sensor, reading):
         yield gen.moment
         raise gen.Return(req.make_reply("ok", name, strategy, *params))
 
+    @gen.coroutine
+    def _handle_bulk_set_sensor_sampling(self, req, msg):
+        if not msg.arguments:
+            raise FailReply("No sensor name given.")
+
+        # The client connection that is not specific to this request context
+        client = req.client_connection
+
+        def inform_callback(sensor, reading):
+            """Inform callback for sensor strategy."""
+            timestamp, status, value = reading
+            cb_msg = format_inform_v5(sensor, timestamp, status, value)
+            client.inform(cb_msg)
+
+        names_arg = ensure_native_str(msg.arguments[0])
+        names = names_arg.split(',')
+
+        if not len(msg.arguments) > 1:
```

Comment:
Would it be easier to read if removed the `not`?
```suggestion
        if len(msg.arguments) <= 1:
```